main.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/Daniorocket/cookit-backend/handlers"
+	"github.com/Daniorocket/cookit-backend/routing"
+)
+
+func main() {
+	port := os.Getenv("PORT")
+	router, err := routing.NewRouter()
+	if err != nil {
+		log.Print("Failed to init Router", err)
+		return
+	}
+	router.Use(handlers.Authenticate)
```

Comment:
Skoro cały router inicjalizuje się w pakiecie Routing, to czemu ten middleware jest ładowany tutaj?

Reply:
Zrobiłem to tak dla przejrzystości kodu(intuicyjnie), ale poprawię